apps/frontend/src/app/Data/Characters/Furina/index.tsx

```diff
@@ -0,0 +1,689 @@
+    ),
+    percent(-dm.passive2.max_interval_dec_)
+  )
+)
+const member_addl: Data = {
+  premod: {
+    skill_dmg_: a4Member_dmg_,
+  },
+}
+
+const [condC2OverstackPath, condC2Overstack] = cond(key, 'c2Overstack')
+const c2OverstackArr = range(50, dm.constellation2.stacks, 50)
+const c2Overstack_hp_ = greaterEq(
+  input.constellation,
+  2,
+  lookup(
+    condC2Overstack,
+    objKeyMap(c2OverstackArr, (stack) =>
+      prod(stack, percent(dm.constellation2.hp_, { fixed: 2 }))
+    ),
+    naught
+  )
+)
+
+const [condc6Path, condc6] = cond(key, 'c6')
+const [condC6PneumaPath, condC6Pneuma] = cond(key, 'c6Pneuma')
+
+const c6_infusion = greaterEqStr(
+  input.constellation,
+  6,
+  equalStr(condc6, 'on', 'hydro')
+)
+const c6_auto_dmgInc = greaterEq(
+  input.constellation,
+  6,
+  equal(
+    condc6,
+    'on',
+    prod(
+      sum(
+        percent(dm.constellation6.auto_dmgInc),
+        equal(condC6Pneuma, 'on', percent(dm.constellation6.pneumaAuto_dmgInc))
+      ),
+      input.total.hp
+    )
+  )
+)
+const c6_normal_dmgInc = infoMut(
+  { ...c6_auto_dmgInc },
+  KeyMap.info('normal_dmgInc')
+)
+const c6_charged_dmgInc = infoMut(
+  { ...c6_auto_dmgInc },
+  KeyMap.info('charged_dmgInc')
+)
+const c6_plunging_dmgInc = infoMut(
+  { ...c6_auto_dmgInc },
+  KeyMap.info('plunging_dmgInc')
+)
+
+const dmgFormulas = {
+  normal: {
+    ...Object.fromEntries(
+      dm.normal.hitArr.map((arr, i) => [
+        i,
+        dmgNode('atk', arr, 'normal', {
+          premod: {
+            normal_dmgInc: c6_normal_dmgInc,
+          },
+        }),
+      ])
+    ),
+    thornBladeDmg: dmgNode('atk', dm.normal.bladeThornDmg, 'normal', {
+      hit: { ele: constant(data_gen.ele) },
+    }),
+  },
+  charged: {
+    dmg: dmgNode('atk', dm.charged.dmg, 'charged', {
+      premod: {
+        charged_dmgInc: c6_charged_dmgInc,
+      },
+    }),
+  },
+  plunging: Object.fromEntries(
+    Object.entries(dm.plunging).map(([key, value]) => [
+      key,
+      dmgNode('atk', value, 'plunging', {
+        premod: {
+          plunging_dmgInc: c6_plunging_dmgInc,
+        },
+      }),
+    ])
+  ),
+  skill: {
+    bubbleDmg: dmgNode('hp', dm.skill.bubbleDmg, 'skill'),
+    usherDmg: dmgNode(
+      'hp',
+      dm.skill.usherDmg,
+      'skill',
+      member_addl,
+      member_mult_
+    ),
+    chevalDmg: dmgNode(
+      'hp',
+      dm.skill.chevalDmg,
+      'skill',
+      member_addl,
+      member_mult_
+    ),
+    crabDmg: dmgNode(
+      'hp',
+      dm.skill.crabDmg,
+      'skill',
+      member_addl,
+      member_mult_
+    ),
+    streamsHeal: healNodeTalent(
+      'hp',
+      dm.skill.streamsHealBase,
+      dm.skill.streamsHealFlat,
+      'skill'
+    ),
+  },
+  burst: {
+    skillDmg: dmgNode('hp', dm.burst.skillDmg, 'burst'),
+  },
+  passive1: {
+    heal: greaterEq(input.asc, 1, healNode('hp', dm.passive1.heal, 0)),
+  },
+  passive2: {
+    member_dmg_: a4Member_dmg_,
+  },
+  constellation6: {
+    c6_normal_dmgInc,
```

Comment:
Should we make her C6 NAs be separate for multi opt? Kind of clunky, but might be needed for C6 multi opts

Reply:
Yeah, ig so.

Reply:
eh nvm who's gonna use non-hp scaling uninfused autos